src/main/java/hibernate/controller/AuthenticationController.java

```diff
@@ -0,0 +1,25 @@
+package hibernate.controller;
+
+import hibernate.model.dto.UserRequestDto;
+import hibernate.service.AuthenticationService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/register")
+public class AuthenticationController {
+    private AuthenticationService authenticationService;
```

Comment:
```suggestion
    private final AuthenticationService authenticationService;
```
let's add `final` to all autowired fields in controllers, services, etc.